frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -231,6 +232,9 @@ final case class Project(
             )
             Properties.javaVersion
         }
+      } else if (rtJar.exists) {
+        // jdk 8 doesn't have `release` file
```

Comment:
Och, I actually checked mine and it had it, which distro are you using?

Reply:
Probably it's from [opejdk-ppa](https://launchpad.net/~openjdk-r/+archive/ubuntu/ppa) - version "1.8.0_312".